data/database.js

```diff
@@ -1,6 +1,7 @@
 const ScoutUser = require('./models/ScoutUser');
 const AudioFileLocation = require('./models/AudioFileLocation');
 const Hostname = require('./models/Hostname');
+const ApiUser = require('./models/ApiUser');
```

Comment:
Change to `require('./models/APIUser');`

Reply:
fixed